include/_mgp.hpp

```diff
@@ -236,6 +238,56 @@ inline mgp_type *type_nullable(mgp_type *type) { return MgInvoke<mgp_type *>(mgp
 
 // mgp_graph
 
+inline bool CreateLabelIndex(mgp_graph *graph, const std::string_view label) {
+  return MgInvoke<int>(mgp_create_label_index, graph, std::string(label).c_str());
+}
+
+inline bool DropLabelIndex(mgp_graph *graph, const std::string_view label) {
+  return MgInvoke<int>(mgp_drop_label_index, graph, std::string(label).c_str());
+}
+
+inline mgp_list *ListAllLabelIndices(mgp_graph *graph, mgp_memory *memory) {
+  return MgInvoke<mgp_list *>(mgp_list_all_label_indices, graph, memory);
+}
+
+inline bool CreateLabelPropertyIndex(mgp_graph *graph, const std::string_view label, const std::string_view property) {
+  return MgInvoke<int>(mgp_create_label_property_index, graph, std::string(label).c_str(),
+                       std::string(property).c_str());
```

Comment:
Are you working here with temporary, I know it is extended, but if someone removes const char* and uses just char*, we are all aboard bug train

Reply:
I think this is not a bug but it is better to avoid creating temporary :+1: will fix